diagnostic.md

```diff
@@ -23,7 +26,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$("li.parent.grandparent.child.form")
```

Comment:
Careful here, to chain methods on you have to list them separately.

`$('li').parent().parent().find('form')`